native/src/components/TtsPlayer.tsx

```diff
@@ -0,0 +1,344 @@
+  background-color: ${props => props.theme.colors.grayBackgroundColor};
+  border-radius: 28px;
+  width: 85%;
+  display: flex;
+  flex-direction: ${props => (props.$isPlaying ? 'column' : 'row')};
+  justify-content: center;
+  align-items: center;
+  align-self: center;
+  padding: 6px;
+  position: absolute;
+  bottom: 5px;
+  min-height: 93px;
+  gap: ${props => (props.$isPlaying ? '5px;' : '10px')};
+`
+
+const StyledPanel = styled.View`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 20px;
+`
+
+const StyledPlayIcon = styled(IconButton)`
+  background-color: ${props => props.theme.colors.textColor};
+  width: 50px;
+  height: 50px;
+  border-radius: 50px;
+`
+
+const StyledBackForthButton = styled.TouchableOpacity`
+  display: flex;
+  flex-direction: row;
+  gap: 5px;
+  align-items: flex-end;
+`
+
+const PlayButtonIcon = styled(Icon)`
+  color: ${props => props.theme.colors.grayBackgroundColor};
+`
+
+const StyledNoSoundIcon = styled(Icon)`
+  height: 18px;
+  width: 18px;
+`
+
+const FlipView = styled.View`
+  transform: scaleX(-1);
+`
+
+const StyledText = styled(Text)`
+  font-weight: bold;
+`
+
+const StyledPlayerHeaderText = styled(Text)`
+  font-weight: 600;
+  align-self: center;
+  font-size: 18px;
+`
+
+const CloseButton = styled.TouchableOpacity`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  border-radius: 7px;
+  background-color: ${props => props.theme.colors.themeColor};
+  padding: 5px;
+  gap: 5px;
+  width: 176px;
+`
+
+const CloseView = styled.View`
+  flex-direction: column;
+  gap: 10px;
+`
+
+export type ttsContextType = {
+  content: string | null
+  setContent: React.Dispatch<React.SetStateAction<string | null>>
+  sentenceIndex: number
+  setSentenceIndex: React.Dispatch<React.SetStateAction<number>>
+  visible: boolean
+  setVisible: React.Dispatch<React.SetStateAction<boolean>>
+  title: string
+  setTitle: React.Dispatch<React.SetStateAction<string>>
+  volume: number
+  setVolume: React.Dispatch<React.SetStateAction<number>>
+}
+
+/* eslint-disable @typescript-eslint/no-empty-function */
+export const ttsContext = createContext<ttsContextType>({
+  content: null,
+  setContent: () => {},
+  sentenceIndex: 0,
+  setSentenceIndex: () => {},
+  visible: false,
+  setVisible: () => {},
+  title: '',
+  setTitle: () => {},
+  volume: 50,
+  setVolume: () => {},
+})
+/* eslint-enable @typescript-eslint/no-empty-function */
+
+type TtsPlayerProps = {
+  children: ReactElement
+  initialVisibility?: boolean
+}
+
+const TtsPlayer = ({ initialVisibility = false, children }: TtsPlayerProps): ReactElement => {
+  const { languageCode } = useContext(AppContext)
+  const { t } = useTranslation('layout')
+  const [isPlaying, setIsPlaying] = useState(false)
+  const [expandPlayer, setExpandPlayer] = useState(false)
+  const defaultVolume = 50
+  const [volume, setVolume] = useState(defaultVolume)
```

Comment:
i think you can initialize this with the value from your context, no?
`{volume} = useTtsPlayer()`